pandas/tests/test_resample.py

```diff
@@ -1173,27 +1173,20 @@ def test_resample_frame_basic(self):
         df.resample('M', kind='period').mean()
         df.resample('W-WED', kind='period').mean()
 
-    def test_resample_loffset(self):
+    @pytest.mark.parametrize('loffset', [timedelta(minutes=1),
```

Comment:
if you want to use hypothesis would be great here :> (can be followup, not sure how easy)

Reply:
@jreback 

There are no many examples in pandas codebase how to use hypothesis. Furthermore, I know only 4 cases which need to test: `timedelta(minutes=1), '1min', Minute(1), np.timedelta64(1, 'm')`.  Did I miss something?

Reply:
ping @jreback